waypoints-special/CAN_PG_N-Z.cup

```diff
@@ -0,0 +1,317 @@
+name,code,country,lat,lon,elev,style,rwdir,rwlen,freq,desc
+"05Arthur AP",05ARTHUR,CA,4350.100N,08026.200W,472.4m,2,310,1219.0m,"123.400","Start Point, Finish Point, Turn Point"
+"08Brantford AP",08BRNTFR,CA,4307.683N,08020.850W,259.1m,5,250,1524.0m,"122.825","100 ft wide"
+"11Cambridge",11CMBRDG,CA,4321.583N,08017.517W,297.2m,1,,,,"Shopping Centre"
+"21Elmira AP",21ELMIRA,CA,4335.500N,08030.733W,350.5m,5,270,549.0m,"123.200","90 ft wide"
+"24Fergus AP",24FERGUS,CA,4344.517N,08023.117W,426.7m,2,220,661.0m,"123.400","Turn Point"
+"36Listowel AP",36 LIST,CA,4340.900N,08059.933W,372.8m,2,40,549.0m,"122.800","Turnpoint"
```

Comment:
Consistency in the description (applies to whole file) Also "turnpoint" is confusing, as the type 2 actually signifies airfield.